pkg/asset/machines/machineconfig/powersmt.go

```diff
@@ -0,0 +1,70 @@
+package machineconfig
+
+import (
+	"fmt"
+
+	ignutil "github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
```

Comment:
is 3.2 right here? I know the MCO just added support for 3.4 recently I believe.

Reply:
Currently all other machineconfigs in the installer are still using 3.2 types. I will keep it here for now but thanks for the info!!